ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -275,11 +280,63 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+        )
+        gui_parser.add_argument(
+            "-H",
```

Comment:
```suggestion
            "-h",
```

Reply:
Shouldn't this remain `-H` in order to prevent conflict with `-h` for help?

Reply:
Yeah you're right, I misremembed `-h` as being more consistent with other server CLIs we've written. `-H` is fine.